semperfiwebdesign/all-in-one-seo-pack
# Fix conflict between Chromium v77.0.3865.75 and Gutenberg

Issue #2914

## Proposed changes

Fixes a conflict between Chromium browsers and the Gutenberg editor that causes meta boxes to overlap with the content.

The PR also includes Grunt fixes which are needed for 3.2.8; else Travis CI will fail.

## Types of changes

- Bugfix (non-breaking change which fixes an issue)

## Checklist


## Testing instructions

Install a browser based on Chromium v77.0.3865.75 as well as All in One SEO Pack from WordPress.org. You'll notice that our meta box overlaps with the content. Next, install this PR and confirm that it fixed this.

## Further comments

While this isn't an issue we caused, users thinks we did, so that's why we're adding this fix.
